x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -317,6 +317,7 @@ export const TIMELINE_PREPACKAGED_URL = `${TIMELINE_URL}/_prepackaged` as const;
 export const NOTE_URL = '/api/note' as const;
 export const PINNED_EVENT_URL = '/api/pinned_event' as const;
 export const SOURCERER_API_URL = '/internal/security_solution/sourcerer' as const;
+export const RISK_SCORE_DEPRECATION_API_URL = '/internal/risk_score/deprecation' as const;
```

Comment:
Make a note for myself that we might want to unify the bath path for all the risk_score api. As I called it `/internal/risky_score` in my pr lol